src/raiden_libs/cli.py

```diff
@@ -100,10 +100,15 @@ def call_with_common_options_initialized(**params: Any) -> Callable:
             try:
                 setup_logging(params.pop("log_level"))
                 if not params["state_db"]:
-                    chain_id = str(params["web3"].net.version)
+                    # only RC has `chain_id`, MS and PFS have `web3` object
+                    chain_id = str(params.get("chain_id") or params["web3"].net.version)
```

Comment:
```suggestion
                    chain_id = str(params.pop("chain_id", params["web3"].net.version))
```

Then you should not need the extra pop below.

Reply:
Nice, I expected this but didn't find it in any of the first google results.

Reply:
So actually this doesn't work, as it eagerly evaluated and would complain about `web`  not being a valid key.

Reply:
Well, then you could still do
```python
chain_id = str(params.pop("chain_id", None) or params["web3"].net.version)
```

Reply:
Yes, but as we need to delete it as well when the `state-db` is given, I think it's cleaner that way.